src/style.css

```
@@ -1 +1,72 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: inline-block;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0 0 0 auto;
+  list-style: none;
+  padding: 0;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-right: 50px;
+  height: 60px;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.logo {
+  margin: 10px 0 10px 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__link {
+  display: inline-block;
+  font-weight: 500;
+  text-decoration: none;
+  color: black;
+}
+
+.is-active {
+  position: relative;
+}
+
+.nav__link:hover,
+.is-active,
+.is-active:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  display: block;
+  content: "";
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  position: absolute;
+  margin-top: 18px;
```

```suggestion
  bottom: 0;
```
Не зрозуміло, звідки взялась цифра 18. Старайся уникати таких "магічних чисел"
Зпозиціонуй цей елемент в самому низу батьківського (використовуй bottom: 0)